conf/broker.conf

```diff
@@ -1966,3 +1966,7 @@ brokerInterceptors=
 
 # Enable or disable the broker interceptor, which is only used for testing for now
 disableBrokerInterceptors=true
+
+# If automatic topic creation is enabled and the name of the topic contains 'cluster',
+# the topic cannot be automatically created.
+allowAutoTopicCreationWithLegacyNamingScheme=false
```

Comment:
It's better to put this config after `allowAutoTopicCreationType`. These configs are grouped by use cases in this file.